fcrepo4/fcrepo4
# Upgrade JQuery to 1.12.4

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2578

# What does this Pull Request do?
Upgrades JQuery to 1.12.4. 

# What's new?
Hopefully nothing.  This is not the most recent version of JQuery, but is free of known
vulnerabilities.  Using more recent versions (3.0+) result in errors.

Upgraded HttpUnit, as it did not have featured required by JQuery
1.12.

Also upgraded HttpClient and HttpCore, as the latest HttpUnit required
newer versions of these to function.

# How should this be tested?
Verify that it builds `mvn clean install`.  Then try it out and see if the UI still works.
in `fcrepo-webapp` do `mvn install -Pone-click`, and run the `*-jetty-console.jar`. 

# Interested parties
@fcrepo4/committers
